nessie/src/test/java/org/apache/iceberg/nessie/TestCustomNessieClient.java

```diff
@@ -46,7 +46,7 @@ public void testNoCustomClient() {
         "nessie",
         ImmutableMap.of(
             CatalogProperties.WAREHOUSE_LOCATION,
-            temp.toUri().toString(),
```

Comment:
This was correct before. Location should be a full URI including the scheme.

Reply:
ok. I have updated now.  I hope it is ok now.